### DISCUSSION TITLE
What are the chances of adding comics and manga lists?

### DISCUSSION THREAD

**Discussion by ExtraFroggy** [Crossoufire/MyLists#24] [Feature Requests]
What are the chances of adding comics and manga lists?

**Comment by vincedelmas** [comment]
Hello,

I think it would be possible to implement the MangaList using the Jikan API (which itself uses MyAnimeList). However, the API is “authentication-free”, and only allows a max of 3 requests/sec or 60 requests/minute. This could be a very limiting. We also need to decide whether Manga should have its own separate list or be added into the existing BooksList. There are already 5 types of lists, and adding more might make things a bit too complex. So maybe we could include Manga within the BooksList and add more filtering options ?

For the comics, I don't know any free API for this. But if we find one, I think Comics should also be added into the BooksList to keep the profile interface streamlined and avoid too many separate lists.

What do you think?

**Reply by ExtraFroggy** [reply]
That would be great, but I think manga and comics should definitely have their own separate lists since they’re very different from each other and not the same as books. For a book list, you can track how many pages you’ve read, but for manga, it makes more sense to track how many chapters you’ve read.
